app/controllers/admin/api_controller.rb

```diff
@@ -0,0 +1,61 @@
+class Admin::ApiController < ApplicationController
+  before_filter :admin_only
+  before_filter :check_for_cancel, only: [:create, :update]
+
+  def index
+    @api_keys = if params[:query]
+                  sql_query = "%" + params[:query] + "%"
+                  ApiKey.where("name LIKE ?", sql_query)
+                        .order("name")
+                        .paginate(page: params[:page])
```

Comment:
This is probably a good idea -- I, um, broke tag the other day because I didn't put my punctuation on the previous line. /o\ (Sorry for nagging if you were going to do this but just haven't had the time yet! I saw you'd merged with master recently, so I just wanted to make sure.)